google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/AppEngineCloudType.java

```diff
@@ -202,6 +203,16 @@ public void setName(String name) {
         this.name = name;
       }
 
+      @Nullable
+      @Override
+      public File getFile() {
+        return new File(userSpecifiedFilePath);
```

Comment:
I think if the userSpecifiedFilePath is not set then the fix that @chanseokoh is working on (#560) should prevent the custom deployment UI from being shown in the first place.